### ISSUE TITLE
Evaluation

### ISSUE THREAD

**Issue by stefan-it** [clarinsi/janes-ner#2]
Hi @nljubesi,

thanks for providing this repository and data.

I would like to train a NER models for the different languages. Unfortunately, there's no "official" dataset split into training, development and test set. Could you give any advice of how to compare results with the reported F1-score in the main `README.md`.

Did you use cross validation (-> what was the split size?) for that 🤔

Thanks many in advance :heart:

**Comment by nljubesi** [comment]
Hi, @stefan-it.

What we did was just to split into train and test as no hyperparameter tuning was necessary.

For this very basic evaluation we split the ssj500k (http://hdl.handle.net/11356/1210) and the janestag (http://hdl.handle.net/11356/1123) datasets by including the first 80% of sentences into train and the last 20% into test.

We do have a benchmarking repo set up (https://github.com/clarinsi/babushka-bench), but we did not manage to climb up to the layer of named entities yet.
